frame/executive/src/lib.rs

```diff
@@ -220,28 +220,28 @@ where
 }
 
 impl<
-		System: frame_system::Config + EnsureInherentsAreFirst<Block>,
-		Block: traits::Block<Header = System::Header, Hash = System::Hash>,
-		Context: Default,
-		UnsignedValidator,
-		AllPalletsWithSystem: OnRuntimeUpgrade
-			+ OnInitialize<System::BlockNumber>
-			+ OnIdle<System::BlockNumber>
-			+ OnFinalize<System::BlockNumber>
-			+ OffchainWorker<System::BlockNumber>,
-		COnRuntimeUpgrade: OnRuntimeUpgrade,
-	> Executive<System, Block, Context, UnsignedValidator, AllPalletsWithSystem, COnRuntimeUpgrade>
-where
-	<System as frame_system::Config>::BlockNumber: AtLeast32BitUnsigned,
-	Block::Extrinsic: IdentifyAccountWithLookup<Context, AccountId = System::AccountId>
+	System: frame_system::Config + EnsureInherentsAreFirst<Block>,
+	Block: traits::Block<Header = System::Header, Hash = System::Hash>,
+	Context: Default,
+	UnsignedValidator,
+	AllPalletsWithSystem: OnRuntimeUpgrade
+	+ OnInitialize<System::BlockNumber>
+	+ OnIdle<System::BlockNumber>
+	+ OnFinalize<System::BlockNumber>
+	+ OffchainWorker<System::BlockNumber>,
+	COnRuntimeUpgrade: OnRuntimeUpgrade,
+> Executive<System, Block, Context, UnsignedValidator, AllPalletsWithSystem, COnRuntimeUpgrade>
+	where
+		<System as frame_system::Config>::BlockNumber: AtLeast32BitUnsigned,
```

Comment:
could you try running `cargo fmt`. We try to keep code formatted as it simplifies following migrations